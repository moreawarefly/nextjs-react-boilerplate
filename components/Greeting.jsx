import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.main`
  height: 90vh;
  display: flex;
  justify-content: center;
  align-items: center;
`;

const Greeting = () => (
  <Wrapper>
    Hi :3
  </Wrapper>
);

export default Greeting;
