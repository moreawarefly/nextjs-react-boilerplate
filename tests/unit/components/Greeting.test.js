import renderer from 'react-test-renderer';
import Greeting from '../../../components/Greeting';

it('renders correctly', () => {
  const tree = renderer
    .create(<Greeting />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
