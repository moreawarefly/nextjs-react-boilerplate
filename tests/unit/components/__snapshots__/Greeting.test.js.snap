// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  className="sc-bdVaJa bhqfqN"
>
  Hi :3
</main>
`;
